### ISSUE TITLE
Failed to pick a color on Ubuntu 22.04

### ISSUE THREAD

**Issue by dawidd6** [FineFindus/eyedropper#6]
**Describe the bug**
I've installed eyedropper through flatpak on Ubuntu 22.04, run it, clicked on "picker" icon and got:
> Failed to pick a color
at bottom of app's window.

**To Reproduce**

1. Be on Ubuntu 22.04 with flatpak installed from default apt repos
2. Execute `flatpak install --user flathub com.github.finefindus.eyedropper`
3. Execute `flatpak run com.github.finefindus.eyedropper`
4. Try to pick a color
5. See error

**Expected behavior**
A color is picked from the screen.

**Screenshots**
![image](https://user-images.githubusercontent.com/9713907/190986279-5f4bc414-e8b4-4c61-91d7-aefd6dff43f8.png)

**Information**

- OS: Ubuntu 22.04
- Desktop Environment: GNOME 42 (X11)
- GTK-Theme/Adwaita-Style: Yaru-dark
- Version: 0.1.0

**Additional context**
> Mon 19 Sep 11:12:35 ➜  ~  flatpak run com.github.finefindus.eyedropper 
 ERROR eyedropper::window > Portal request failed: org.freedesktop.zbus.Error: No such interface “org.freedesktop.portal.Screenshot” on object at path /org/freedesktop/portal/desktop

**Comment by dawidd6** [comment]
Just tried `org.kryogenix.Pick`, similar app and it works just fine.

**Comment by dawidd6** [comment]
I have them installed. But I'm on X11 session, not Wayland.

**Comment by FineFindus** [comment]
I don't think this is a Wayland/X11 issue. But just to confirm, I tried it on X11 as well and could not reproduce it.

I will have to look into this. Did you modify some part of your system (xdg-portals, etc.), that might be related to this?

**Comment by dawidd6** [comment]
No, actually I don't even have enough knowledge about portals to even think about modifying something related to them. Do you have flatpak installed from PPA or official Ubuntu repo? Maybe there is some bug that is fixed in newer releases as I use the one from Ubuntu repo.

**Comment by FineFindus** [comment]
I followed the official guide, using the Ubuntu repo.

I will try to replicate it in a fresh Ubuntu Install. Alternatively, we could try to use a normal, non-flatpak, binary to see if the issue might be flatpak related. 
I'm sorry that I can't help you like I would want to, but it is really hard to fix an issue that I can't replicate myself.

**Comment by dawidd6** [comment]
It's okay, I get it. I could try the binary if you provide one, sure.

**Comment by dawidd6** [comment]
Tried to install latest flatpak build from CI run and it is the same FYI.

**Comment by dawidd6** [comment]
I'll try on my second machine with similar system setup and report back.

**Comment by dawidd6** [comment]
Weird, it works no problem after a reboot on both my machines. So I guess it was some problem on my side. Thanks for assistance, closing.

**Comment by CSpencerND** [comment]
Same thing happening to me on fedora.
"Failed to pick a color."
`ERROR eyedropper::window > Portal request didn't succeed: Other`

**Comment by FineFindus** [comment]
This seems to be a different bug. Could you please create a new issue for this?

**Comment by jjrob** [comment]
Linux noob here. Thanks for developing Eyedropper. I reproed this problem on Linux Mint 21.1 Cinnamon. The machine is up to date with all the latest updates available through the Mint Update Manager. I installed the Eyedropper flatpak with the Mint Software Manager. Version info in attached screenshot. After installing, I clicked Pick A Color and got the "Failed to pick a color" prompt, as shown in the screenshot. Rebooting did not fix it.

![Eyedropper](https://user-images.githubusercontent.com/12236262/232068932-1a78b976-8bba-4309-bbe4-737a4b72e84c.png)
